package.json

```diff
@@ -17,13 +17,14 @@
   },
   "devDependencies": {
     "@types/node": "^22.10.6",
-    "@wdio/cli": "^8.41.0",
-    "@wdio/local-runner": "~8.41.0",
-    "@wdio/mocha-framework": "^8.41.0",
-    "@wdio/spec-reporter": "8.41.0",
-    "@wdio/testingbot-service": "^8.41.0",
+    "@wdio/cli": "^9.5.7",
+    "@wdio/local-runner": "~9.5.7",
+    "@wdio/mocha-framework": "^9.5.0",
+    "@wdio/spec-reporter": "9.5.0",
+    "@wdio/testingbot-service": "^9.5.7",
     "ts-mocha": "^10.0.0",
-    "ts-node": "^10.9.2"
+    "ts-node": "^10.9.2",
+    "@wdio/types": "^9.5.0"
```

Comment:
[Nitpick] Could you move this up? In our node.js repos I usually expect the packages to be sorted alphabetically which happens automatically when you use `npm install`. This addition looks like you added it manually